ports/servoshell/desktop/minibrowser.rs

```diff
@@ -175,36 +183,39 @@ impl Minibrowser {
             if window.fullscreen().is_none() {
-                TopBottomPanel::top("toolbar").show(ctx, |ui| {
+                let frame = egui::Frame::default()
+                    .fill(Color32::from_gray(32))
+                    .inner_margin(4.0);
+                TopBottomPanel::top("toolbar").frame(frame).show(ctx, |ui| {
                     ui.allocate_ui_with_layout(
                         ui.available_size(),
                         egui::Layout::left_to_right(egui::Align::Center),
                         |ui| {
-                            if ui.button("back").clicked() {
+                            if ui.add(Minibrowser::toolbar_button("⏴")).clicked() {
                                 event_queue.borrow_mut().push(MinibrowserEvent::Back);
                             }
-                            if ui.button("forward").clicked() {
+                            if ui.add(Minibrowser::toolbar_button("⏵")).clicked() {
                                 event_queue.borrow_mut().push(MinibrowserEvent::Forward);
                             }
+
+                            match self.load_status {
+                                LoadStatus::LoadStart | LoadStatus::HeadParsed => {
+                                    if ui.add(Minibrowser::toolbar_button("X")).clicked() {
+                                        warn!("Do not support stop yet.");
+                                    }
+                                },
+                                LoadStatus::LoadComplete => {
+                                    if ui.add(Minibrowser::toolbar_button("↻")).clicked() {
+                                        event_queue.borrow_mut().push(MinibrowserEvent::Reload);
+                                    }
+                                },
+                            }
+                            ui.add_space(2.0);
+
                             ui.allocate_ui_with_layout(
                                 ui.available_size(),
                                 egui::Layout::right_to_left(egui::Align::Center),
                                 |ui| {
-                                    if ui.button("go").clicked() {
-                                        event_queue.borrow_mut().push(MinibrowserEvent::Go);
-                                        location_dirty.set(false);
-                                    }
-
-                                    match self.load_status {
-                                        LoadStatus::LoadStart => {
-                                            ui.add(Spinner::new().color(Color32::GRAY));
-                                        },
-                                        LoadStatus::HeadParsed => {
-                                            ui.add(Spinner::new().color(Color32::WHITE));
-                                        },
-                                        LoadStatus::LoadComplete => { /* No Spinner */ },
-                                    }
```

Comment:
this change removes the spinner, was that intentional? do you have any thoughts on how we can indicate that a page is loading?